toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/SceneView.kt

```diff
@@ -165,6 +169,7 @@ private fun SceneViewEventHandler(
     sceneView: SceneView,
     onNavigationChanged: ((isNavigating: Boolean) -> Unit)?,
     onInteractingChanged: ((isInteracting: Boolean) -> Unit)?,
+    onSpatialReferenceChanged: ((spatialReference: SpatialReference?) -> Unit)? = null,
```

Comment:
we shouldn't provide default value here, 
```suggestion
    onSpatialReferenceChanged: ((spatialReference: SpatialReference?) -> Unit)?,
```